tasks/broad/Alignment.wdl

```diff
@@ -61,52 +61,43 @@ task SamToFastqAndBwaMemAndMba {
 
     # set the bash variable needed for the command-line
     bash_ref_fasta=~{reference_fasta.ref_fasta}
-    # if reference_fasta.ref_alt has data in it,
-    if [ -s ~{reference_fasta.ref_alt} ]; then
```

Comment:
Quick question, do we know why this check was included previously?  Was there a common problem happening with missing alt files?  Are we now going to need to actively watch out for that again, instead of relying on this check?

Reply:
Good question. Maybe @kbergin knows?

Reply:
I’m not sure- @gbggrant do you recall?

Reply:
I think one of these calls falls over if there is no ref_alt file available

Reply:
I had thought it was because there was not a ref_alt for one of the references

Reply:
hg19 has a 0 size reference file, so this probably was avoiding that issue
```0 B  2019-12-06T23:55:15Z  gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.alt```